### ISSUE TITLE
Flowers Classification

### ISSUE THREAD

**Issue by abhisheks008** [abhisheks008/DL-Simplified#14]
### Deep Learning Simplified Repository (Proposing new issue)
:red_circle: **Project Title** : Flowers Classification</br>
:red_circle: **Aim** : Build a model based on Tensorflow to classify the flowers from the given dataset.</br>
:red_circle: **Dataset** : https://www.kaggle.com/datasets/ryanholbrook/tensorflow-flowers</br>
:red_circle: **Approach** : Try to use 3-4 algorithms to implement the models and compare all the algorithms to find out the best fitted algorithm for the model by checking the accuracy scores. Also do not forget to do a exploratory data analysis before creating any model.

**********************************************************************************
### 📍 **Follow the Guidelines to Contribute in the Project :**
* You need to create a separate folder named as the Project Title.
* Inside that folder, there will be four main components.
   * Images - To store the required images.
   * Dataset - To store the dataset or, information/source about the dataset.
   * Model - To store the machine learning model you've created using the dataset.
   * `requirements.txt` - This file will contain the required packages/libraries to run the project in other machines.
* Inside the `Model` folder, the `README.md` file must be filled up properly, with proper visualizations and conclusions.

***********************************************************************
:red_circle::yellow_circle: **Points to Note :**

- The issues will be assigned on a first come first serve basis, 1 Issue == 1 PR.
- "Issue Title" and "PR Title should be the same. Include issue number along with it.
- Follow Contributing Guidelines & Code of Conduct before start Contributing.

***********************************************************************
:white_check_mark: **To be Mentioned while taking the issue :**
- Full name : 
- GitHub Profile Link : 
- Email ID :
- Participant ID (if applicable):
- Approach for this Project :
- What is your participant role? (Mention the Open Source program)

*************************************************************
Happy Contributing 🚀 

All the best. Enjoy your open source journey ahead. 😎

**Comment by shwetmala55** [comment]
Full name :Shwetmala Jha
GitHub Profile Link : https://github.com/shwetmala55
Email ID : shwetmala02@gmail.com
Approach for this Project :classification of flowers
What is your participant role?-contributor ,SSOC-2022

**Comment by abhisheks008** [comment]
Can you elaborate your deep learning approaches? @shwetmala55

**Comment by 072arushi** [comment]
@shwetmala55 you have been assigned this issue. All the best.

**Comment by shwetmala55** [comment]
> Can you elaborate your deep learning approaches? @shwetmala55

will use different method to train the model and compare their accuracy, I recently started with deep learning so intend to learn while contributing.

**Comment by shwetmala55** [comment]
> @shwetmala55 you have been assigned this issue. All the best.

Thank you!

**Comment by abhisheks008** [comment]
@shwetmala55 updates please.

**Comment by abhisheks008** [comment]
Updates please @shwetmala55

**Comment by shwetmala55** [comment]
> Updates please @shwetmala55

im sorry for delay.. facing some issues.. plz give me more time to complete this project

**Comment by abhisheks008** [comment]
Okay, take your time @shwetmala55

**Comment by manichander429** [comment]
Full Name:- B Mani Chander
Github Profile link:- https://github.com/manichander429
Email id:- manichander429@gmail.com
Approach for this project:- so recently I developed a cnn model for diabetic retinopathy detection. Coming to the approach First, I will perform data augmentation, so that the model I develop can learn enough features to detect. Then, I would try using different models/architectures like cnn, ResNet, AlexNet, GoogleNet.. Etc. Then I will perform some hypertuning in order to find the optimal hyperparameters and then I will compare the accuracy of the developed models. 
What is your participant role?-contributor ,SSOC-Season-2

**Comment by manichander429** [comment]
Hii @abhisheks008 .. I would like to contribute to this repo
